data/vibe/data/serialization.d

```diff
@@ -1474,14 +1474,14 @@ unittest // Testing corner case: Variadic template constructors and methods
 @safe unittest // Immutable data deserialization
 {
 	import vibe.data.json;
-	
```

Comment:
Unrelated changes due to trailing whitespace - this is the past!
-> You will like https://github.com/rejectedsoftware/vibe.d/pull/1800 :)